src/com/company/Game.java

```diff
@@ -7,6 +7,12 @@
 import java.util.*;
 import java.util.function.Predicate;
 
+/**
+ * The Game class contains the random map generation,
+ * as well as handles the movement of the player
+ * and interaction with the faced objects.
```

Comment:
and their interaction....
or
and interacts with....

Reply:
Just looked at it and it's not a typo but you can change it if you wish! It's read as 'handles ___ interaction with faced objects' but you can specify with their if needed